### PR TITLE
[ADD] sale_documents_comments renamed

### PR DESCRIPTION
**Pull request by chienandalu**
Related https://github.com/OCA/sale-workflow/pull/509#issuecomment-317485138

cc @Tecnativa

### CODE REVIEW

File: openerp/addons/openupgrade_records/lib/apriori.py [hunk 1]
```diff
@@ -29,6 +29,7 @@
     'marketing_security_group': 'mass_mailing_security_group',
     # odoomrp/odoomrp-wip
     'product_variants_no_automatic_creation': 'product_variant_configurator',
+    'sale_documents_comments': 'sale_comment_propagation',
```
Comment (MiquelRForgeFlow) [inline openerp/addons/openupgrade_records/lib/apriori.py]: the module https://github.com/OCA/sale-workflow/pull/509 is still named `sale_comment_propagation`, see last commit

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

**Comment by mreficent** [comment]
I think this PR should be closed, at least while the referenced PR is named as it is now

File: openerp/addons/openupgrade_records/lib/apriori.py (hunk 1 above)
Comment (pedrobaeza) [reply openerp/addons/openupgrade_records/lib/apriori.py]: We know that. This is for ensuring the migration path from odoomrp-wip module to OCA. If you have already 8.0 OCA module, this is harmless (well, not yet).

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
@chienandalu, you have to include the renaming in the base module where update_module_names is called with merge_modules=True option, as you can have both modules (old and new) in your module list.

File: openerp/addons/openupgrade_records/lib/apriori.py (hunk 1 above)
Comment (pedrobaeza) [reply openerp/addons/openupgrade_records/lib/apriori.py]: It has been already done in other odoomrp-wip modules. See https://github.com/OCA/OpenUpgrade/blob/3243e327e903bd0b24db41f9cf922bc2cdb7fc20/openerp/addons/openupgrade_records/lib/apriori.py#L30

Comment (MiquelRForgeFlow) [reply openerp/addons/openupgrade_records/lib/apriori.py]: Oh, fine!

**Comment by chienandalu** [comment]
Done @pedrobaeza

**Comment by pedrobaeza** [comment]
But you haven't removed it from apriori.py...

**Comment by chienandalu** [comment]
Sure, @pedrobaeza... :sweat_smile:  It's done

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
@chienandalu, why you put the depends in the head comment? It doesn't depend at all.

**Comment by chienandalu** [comment]
@mreficent Yes, now it doesn't. Changed.

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
It seems Travis has fixed its problem referencing $TRAVIS_COMMIT with forced pushed PRs...

**Pull request merged**